db/migrate/20230307130502_add_extension_module_to_subjects.rb

```diff
@@ -0,0 +1,5 @@
+class AddExtensionModuleToSubjects < ActiveRecord::Migration[7.0]
+  def change
+    add_column :subjects, :extension_module, :boolean
```

Comment:
Actually, subjects under this category are not only `Módulos de Extensión` but also `Módulos de Taller` so naming here is not reeeeeally accurate...

Let me know what you guys think about that 🙂